aspnetcore/fundamentals/minimal-apis/samples/WebMinAPIs/Snippets/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace WebMinAPIs.Snippets;
+
+public static class Program
```

Comment:
Why is there a program main?

Reply:
It's just the class, without a `Main` method. It's a way of including code snippets in the article that always compiles, but isn't really part of the sample. I've used a `Program` class here because it shows where the code would be, as I can't put top level statements outside of the root-level `Program.cs`.